content/channels/index.textile

```diff
@@ -753,13 +753,15 @@ channel.Publish(context.Background(), "action", "boom!")
 
 h3(#idempotency). Idempotent publish
 
-Idempotency ensures that multiple publishes of the same message cannot result in duplicate messages.
+By default idempotentRestPublishing is enabled in all latest ably SDKs. It ensures that publishing of a message doesn't result in duplicate messages.
```

Comment:
I think we should be more specific than "in all latest Ably SDKs", can we point to a specific version, or if there are too many to list for different SDKs, we should keep this statement as-is.